src/Controls/src/Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -71,7 +71,13 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 		}
 
 		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='UseSafeAreaProperty']/Docs/*" />
-		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page), false);
+		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create("UseSafeArea", typeof(bool), typeof(Page),		
```

Comment:
Part of me thinks we should just make this true by default on `iOS` also. It seems like not wanting to account for the safe area is the odd one out.

Reply:
yeah i agree think before was different because it was a "new" feature and we didn't want to break existing apps.

Reply:
Chatted with @davidortinau about this one a bit as well and he's all for just defaulting this to `true`